### DISCUSSION TITLE
[@types/facebook-instant-games] priceAmount missing in interface Product

### DISCUSSION THREAD

**Discussion by SnoozingC4t** [DefinitelyTyped/DefinitelyTyped#66284] [Issues with a @types package]
priceAmount is missing in the interface Product
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dcb2b623f115fc1c59d6812df96630768b53fffd/types/facebook-instant-games/index.d.ts#L1286
Documentation:
https://developers.facebook.com/docs/games/instant-games/sdk/fbinstant7.1/

[a comment by an automated account is not shown]

**Comment by doterax** [comment]
Fixed in this PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66286

**Comment by SnoozingC4t** [comment]
thanks!
